src/Compilers/VisualBasic/Test/Semantic/Semantics/UserDefinedBinaryOperators.vb

```diff
@@ -2546,6 +2546,35 @@ End Module
 
         End Sub
 
+
+        <WorkItem(14688, "https://github.com/dotnet/roslyn/issues/14688")>
+        <Fact>
+        Public Sub OperatorsWithDefaultValuesAreNotBound()
+
+            Dim compilationDef =
+<compilation name="OperatorsWithDefaultValuesAreNotBound">
+    <file name="a.vb"><![CDATA[
+Public Class Foo
+    Public Shared Operator Not(Optional x As Foo = Nothing) As Integer 'BIND1:"Foo"
```

Comment:
>Nothing [](start = 51, length = 7)

Please add a test that uses some meaningful default value and verifies SemanticModel behavior within the value.